docs/faq.md

```diff
@@ -172,6 +172,12 @@ Containers:
       --timeout=61s
 ```
 
+## IMDSv2 Support
```

Comment:
I think we should strongly consider including a line in this section suggesting the use of Kubernetes metadata when possible - IMDS metadata requires configuring IMDS in an insecure manner (by increasing the hop limit).

Reply:
Hesitant to provide that recommendation given Kubernetes metadata does not have information about attached ENIs ([NumAttachedENIs](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/85563fbdcb31e7234abf9f9a335008ee0c70c5b8/pkg/cloud/metadata_k8s.go#L82) is hard-coded to 1) and thus the allocatable count as reported by `NodeGetInfo` may be incorrect. That said, can add this recommendation in next revision if you strongly recommend it.

Reply:
Fair enough